wave_vision/src/matcher/brute_force_matcher.cpp

```diff
@@ -84,41 +85,38 @@ void BruteForceMatcher::checkConfiguration(
 
 std::vector<cv::DMatch> BruteForceMatcher::filterMatches(
   std::vector<cv::DMatch> &matches) const {
-    std::vector<cv::DMatch> filt_matches;
-    float min_distance;
-    std::vector<cv::DMatch>::iterator closest_match;
+    std::vector<cv::DMatch> filtered_matches;
 
     // Determine closest match
-    closest_match = std::min_element(matches.begin(), matches.end());
-    min_distance = closest_match->distance;
+    auto closest_match = std::min_element(matches.begin(), matches.end());
+    float min_distance = closest_match->distance;
```

Comment:
`auto` seems right here to avoid any future troubles e.g. if this becomes a `double`